manager/manager.go

```diff
@@ -135,7 +136,12 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, maxHousekeepingIn
 	if err != nil {
 		glog.Warningf("Unable to connect to Docker: %v", err)
 	}
-	context := fs.Context{DockerRoot: docker.RootDir(), DockerInfo: dockerInfo}
+	rktPath, err := rkt.RktPath()
+	if err != nil {
+		glog.Warningf("unable to connect to Rkt api service: %v", err)
```

Comment:
So if we initialise a manager and rkt is unavailable, we just warn and then we're stuck with that for the lifetime of the manager? is that the same behaviour as with docker? surprised it doesn't try to recover...